cime_config/config_grids.xml

```diff
@@ -1517,6 +1517,16 @@
       <mask>ICOS10</mask>
     </model_grid>
 
+    <model_grid alias="ne1024np4_oRRS15to5" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne1024np4</grid>
+      <grid name="lnd">ne1024np4</grid>
+      <grid name="ocnice">oRRS15to5</grid>
+      <grid name="rof">null</grid>
```

Comment:
Sorry, missed the email notice for this PR. null for rof is ok if to run with SROF. However, in SCREAMv0, the F2010-SCREAM-* compsets are using MOSART, it would fail to create a case using this grid as is. 

If change to r0125 for rof, need to add ATM2ROF, LND2ROF maps from ne1024np4_to_r0125.

Reply:
I was wrong. F2010-SCREAM-HR can still be created with null rof grid. And it appears atm2rof and lnd2rof are using idmap, I assume that means rof (MOSART) would also be using ne1024np4. This is still not what we wanted.

Reply:
Continuing to learn of this: likely due to null rof grid, although the mosart component is built into the executable, by default, do_rtm = .false. is set in mosart_in, which effectively turns MOART inactive during runtime.